rune/libenclave/epm/epm.go

```diff
@@ -51,7 +53,9 @@ func GetCache(ID string) *v1alpha1.Enclave {
 	Type := "enclave-cache-pool"
 	cacheResp, err := c.GetCache(ctx, &v1alpha1.GetCacheRequest{Type: Type, ID: ID})
 	if err != nil {
-		logrus.Fatalf("could not get cache from enclave cache pool: %v", err)
+		syscall.Unlink(sockpath)
+		logrus.Warnf("EPM service is not started, please start it firstly!")
```

Comment:
So this warning means "we found epm sock file but the service behind this sock doesn't exist, so print this warning", right? I guess this situation may be caused by:

- intend to create such a sock file without actually launch epm.
- this sock file is not deleted correctly and left since last epm run.

Correct?

Reply:
It doesn't depend on sock file. It's releated with epm service status.
If there is not sock file(epm service down absolutely) or other epm service down, the c.GetCache will return gRPC failure which means that the epm service is not in good running state. Later I will patch one HealthZ interface which will be used to get epm serivce's running state. In epm client,  it will query epm service status by the interface before calling c.GetCache and c.SaveCache

Reply:
So if I intend to manually create epm sock file, what happens?

Reply:
Even manully create epm sock file, epm service is down, when c.GetCache will return "EPM service is not started, please start it firstly!"

Reply:
Sounds reasonable. But current implementation implies this warning is always printed even epm is not deployed. See the job "Run Skeleton v3 Bundle" at https://github.com/alibaba/inclavare-containers/runs/1666371851?check_suite_focus=true

I mean this warning should appear out only when epm is down accidentally after successfully creating the connection over the sock file.

Reply:
ok I considered this situation.  In fact other patch will be PRed which is about health check interface implmentation. 
Anyway I will remove this unreasonable warning in this patchset( It will spend some time handling conflict between patches.)